src/iris/sso.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import importlib
+
+logger = logging.getLogger(__name__)
+
+
+class SSOManager:
+
+    def __init__(self, config):
+        sso_cfg = config.get('sso', {})
+        if not sso_cfg.get('enabled', False):
+            logger.info('SSO passthrough enabled')
```

Comment:
Should this log say disabled instead of enabled?

Reply:
well the passthrough is enabled, but I think you're right that it would be less confusing to just say "SSO disabled"